src/js/components/Page.js

```diff
@@ -23,19 +25,31 @@ var Page = React.createClass({
     ])
   },
 
+  componentWillMount: function () {
+    this.store_listeners = [
```

Comment:
actually since this is the old type of component, you can add it on directly to the object (similar to how you do `mixins: [etc, etc]`